python/tempo/utils.py

```diff
@@ -8,7 +8,7 @@
 from pyspark.sql.dataframe import DataFrame
 
 logger = logging.getLogger(__name__)
```

Comment:
Can we add a quick comment about this environment variable? What are those expected in the databricks environment? (just the key name)

Reply:
This comment has been added. After the interpolation docstring. The PR will be good to go ahead.